spec/biblio.json

```diff
@@ -0,0 +1,5 @@
+{
+    "https://tc39.es/ecma262/": [
```

Comment:
Maybe this should link to the diff for now, so it is not a dead link?

Reply:
Changing it to `https://arai-a.github.io/ecma262-compare/?pr=1668` didn't fix the link. As soon as I open the diff it ignores the `#sec-class-definitions-static-semantics-containsarguments` added by ecmarkup.

Using `https://arai-a.github.io/ecma262-compare/history/PR/1668/94b1af23d6ca075c6635bcd5b96c6450ec580f51` seems to work, though, but will get out of date if new commits are added.

Reply:
I can kind of hack around it by using this biblio entry:
```
{
    "https://arai-a.github.io/ecma262-compare/?pr=1668&id=sec-class-definitions-static-semantics-containsarguments": [
        {"type": "clause", "id": "sec-class-definitions-static-semantics-containsarguments", ... }
    ]
}
```
Since that diff tool uses `id=`.